data/ri/legislature/Roberto-Brtito-01ddc920-772c-4098-b9f8-8704783940e5.yml

```diff
@@ -0,0 +1,27 @@
+id: ocd-person/01ddc920-772c-4098-b9f8-8704783940e5
+name: Roberto Britto
+given_name: Roberto
+family_name: Britto
+email: sen-britto@rilegislature.gov
+image: https://www.rilegislature.gov/senators/Pictures/Britto 2023.jpg
```

Comment:
@bfossen-ce this person's last name "britto" was spelt "brtito" in the pull. I corrected this in the file for line 2 and 4, but I'm not sure if I can change it in the file name [data/ri/legislature/Roberto-Brtito-01ddc920-772c-4098-b9f8-8704783940e5.yml](https://github.com/openstates/people/pull/898/files#diff-68b580a987f7c5b662a95b34cf3ba8640c5f4c6780de7991ec924449f4b677c2)